packages/event-cache/src/EventCache.js

```diff
@@ -64,6 +64,13 @@ const getPastEvents = memoize(
       debug('Skipped loading event from IPFS cache.')
     }
 
+    // Paranoia check.
+    if (!instance.contract || !instance.contract.options.address) {
+      throw new Error(
+        `EventCache.getPastEvents failure. Contract ${this.prefix} missing address!`
```

Comment:
There's no `this` in this function. Use `instance`

Reply:
yep. fixed.